src/Servers/Kestrel/samples/HttpClientApp/HttpClientApp.csproj

```diff
@@ -9,4 +9,9 @@
     <RuntimeHostConfigurationOption Include="System.Net.SocketsHttpHandler.Http3Support" Value="true" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Seems weird to have a sample include shared content, do we do this elsewhere?

Reply:
We also do it in the h2cat sample.
https://github.com/dotnet/aspnetcore/blob/24ea7ef4b753e1d1d56d13ffa4791af242bcefd0/src/Servers/Kestrel/samples/http2cat/http2cat.csproj#L10

Really these are manual test apps, I'm not sure why we keep calling them samples.